src/BlazorWasmSdk/Targets/Microsoft.NET.Sdk.BlazorWebAssembly.6_0.targets

```diff
@@ -542,6 +542,9 @@ Copyright (c) .NET Foundation. All rights reserved.
         Include="@(StaticWebAsset)"
         Condition="'%(StaticWebAsset.AssetTraitName)' == 'JSModule' and '%(StaticWebAsset.AssetTraitValue)' == 'JSLibraryModule' and '%(AssetKind)' != 'Build'" />
 
+      <!-- We remove the extensions since they are added to the list of static web assets but we need to compute the target path for them -->
```

Comment:
Is this comment accurate? It might just be because I don't know this area of the code at all, but it seems strange that we remove something because "we need to compute the target path for them".

Does removing them somehow cause their target paths to get computed?

Reply:
I added a description to the PR, the issue is that we were including the user extensions in this item group and that was causing an incorrect entry to be added to the manifest.

Extensions and JS modules are pre-processed below on the file as we need the target path for them

Reply:
Adding a description to the PR is great but the PR is not where we'd be looking when trying to understand this code in the future. If there's a way of phrasing the comment that's easier to understand that would be preferable 😃 

> Extensions and JS modules are pre-processed below on the file as we need the target path for them

I guess I just don't know why anything about the target path is a relevant thing to have a comment about here. It seems like an unrelated part of the system. But the way the comment in the source is phrased makes it sound causal, i.e., the act of removing them from this itemgroup is what makes the target path get computed later. Perhaps that is true but it's unexpected.

I know this comment is only a small detail and we shouldn't stress about it or delay the PR for it.

Reply:
All files in blazor.boot.json (dlls and so on) only include the filename.

JS initializer and extensions use the URL because they can live "anywhere" not just inside the _framework folder. So for example you can have a js initializer that lives in `_content/library/library.module.lib.js` and we need to be able to load that.

In a similar vein, when you transform the output during publish, you should not be forced to put those files inside _framework, so we need to compute the URL for those assets an include them there. As an example, you can grab all the dlls from _framework concatenate them and put them inside `_bin/bundle.bdl`